client/src/containers/App/index.js

```diff
@@ -18,12 +18,17 @@ class AppContainer extends Component {
     <App
       todoCards={this.state.todoCards}
       addTodoCard={this.addTodoCard}
+      deleteTodoCard={this.deleteTodoCard}
     />
   )
 
   addTodoCard = (card) => {
     this.setState(state => ({ todoCards: [...state.todoCards, card] }))
   }
+
+  deleteTodoCard = (cardId) => {
+    this.setState(state => ({ todoCards: state.todoCards.filter(el => el.id !== cardId) }))
+  }
```

Comment:
@DDKatch https://trello.com/c/xkkmIu2M/40-when-i-develop-the-app-id-like-not-to-see-linter-errors is merged (nothing is changed in linter), so you can go with:

```js
deleteTodoCard = cardId =>
  this.setState(state => ({ todoCards: state.todoCards.filter(el => el.id !== cardId) }))
```
instead of:
```js
deleteTodoCard = (cardId) => {
  this.setState(state => ({ todoCards: state.todoCards.filter(el => el.id !== cardId) }))
}
```